data/releasenotes.json

```
@@ -1,4 +1,25 @@
 [
+		{
+		"version": "v2025-10-22",
+		"date": "2025-10-22",
+		"breaksdb": false,
+		"notes": [
+			"<b>Maintenance:</b> Improve scheduling under high query load to reduce P99 latencies.",
+			"<b>Maintenance:</b> Harden prepared statements against parallel DDL operations",
+			"<b>Maintenance:</b> Detect out of bounds read of optimistic single value data block lookup",
```

I think this sentence is super specific, and maybe hard to understand for people. My suggestion would be sth like (but not 100% happy):
```suggestion
			"<b>Maintenance:</b> Improve detection of invalid data during query execution",
```

---

i changed the working but good point

---

Maybe something like this?

```suggestion
			"<b>Maintenance:</b> Prevent invalid data by always validating optimistic reads before taking conditional actions.",
```

---

I replaced it by "Fix rare cases of out-of-bound reads during query processing" following a suggestion of our marketing department.

---

I like the suggestion of our marketing department